test/HostActivationTests/GivenThatICareAboutStandaloneAppActivation.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Text.RegularExpressions;
+using Xunit;
+using FluentAssertions;
+using Microsoft.DotNet.CoreSetup.Test;
+using Microsoft.DotNet.Cli.Build.Framework;
+
+namespace Microsoft.DotNet.CoreSetup.Test.HostActivation.StandaloneApp
+{
+    public class GivenThatICareAboutStandaloneAppActivation
+    {
+        private static TestProjectFixture PreviouslyBuiltAndRestoredStandaloneTestProjectFixture { get; set; }
+        private static TestProjectFixture PreviouslyPublishedAndRestoredStandaloneTestProjectFixture { get; set; }
+        private static RepoDirectoriesProvider RepoDirectories { get; set; }
+
+        static GivenThatICareAboutStandaloneAppActivation()
+        {
+            RepoDirectories = new RepoDirectoriesProvider();
+
+            var buildFixture = new TestProjectFixture("StandaloneApp", RepoDirectories);
+            buildFixture
+                .EnsureRestored(RepoDirectories.CorehostPackages, RepoDirectories.CorehostDummyPackages)
+                .BuildProject(runtime: buildFixture.CurrentRid)
+                .ReplaceTestProjectOutputHostFromDotnet(buildFixture.BuiltDotnet);
+
+            var publishFixture = new TestProjectFixture("StandaloneApp", RepoDirectories);
+            publishFixture
+                .EnsureRestored(RepoDirectories.CorehostPackages, RepoDirectories.CorehostDummyPackages)
+                .PublishProject(runtime: publishFixture.CurrentRid)
+                .ReplaceTestProjectOutputHostFromDotnet(publishFixture.BuiltDotnet);
+
+            PreviouslyBuiltAndRestoredStandaloneTestProjectFixture = buildFixture;
+            PreviouslyPublishedAndRestoredStandaloneTestProjectFixture = publishFixture;
+        }
+
+        [Fact]
+        public void Running_Build_Output_Standalone_EXE_with_DepsJson_and_RuntimeConfig_Local_Succeeds()
+        {
+            var fixture = PreviouslyBuiltAndRestoredStandaloneTestProjectFixture
+                .Copy();
+            
+            var appExe = fixture.TestProject.AppExe;
+
+            Command.Create(appExe).CaptureStdErr().CaptureStdOut().Execute().Should().Pass();
```

Comment:
Repeated elsewhere when we are running the test app

---
In reply to: [66346050](https://github.com/dotnet/core-setup/pull/97#discussion_r66346050) [](ancestors = 66346050)